less/ovh-angular-sidebar-menu.less

```diff
@@ -283,6 +287,7 @@
                         .item-icon i {
                             font-size   : @SIDEBAR_MENU_LVL1_ICON_FONT_SIZE;
                             opacity     : @SIDEBAR_MENU_OPACITY;
+                            margin-bottom: 6px;
```

Comment:
It's crappy :(

Impossible to deal with `display: inline-block` + `vertical-align: middle` if we don't want to use `flex` now ?

Reply:
IMHO Seems to be related directly from `ovh-manager-font` which has is own glyph height.